deployments/arbitrum/usdc.n/configuration.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "Compound USDC(native)",
+  "symbol": "cUSDCNv3",
+  "baseToken": "USDC",
+  "baseTokenAddress": "0xaf88d065e77c8cC2239327C5EDb3A432268e5831",
+  "baseTokenPriceFeed": "0x50834F3163758fcC1Df9973b6e91f0F0F0434aD3",
```

Comment:
This is the same price feed that we're using for the current, bridged version of USDC. Do you know if this price feed will reflect the price of the bridged version or the native version?

Reply:
Yeah unfortunately, I didn't see a new pricefeed from USDC native on Chainlink yet :( 
https://docs.chain.link/data-feeds/price-feeds/addresses?network=arbitrum
I can update to the latest pricefeed once Chainlink has added the USDC native pricefeed. 


Part of me is a bit unsure that if it's ok to use the same pricefeed since those USDC.e and USDC Native are in a sense kind of interchangeable?  (USDC.e bridge back to source chain and burn, then can always mint USDC Native at Arbitrum 1:1).

Reply:
Interesting to think about because they are different in some sense. But I think the existing `USDC / USD` price feed makes more sense for native USDC than USDC.e since it is the canonical USDC.